dev-python/cadquery-ocp/cadquery-ocp-7.5.2_rc20210706.ebuild

```diff
@@ -0,0 +1,160 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8..9} )
+
+# The "cmake" eclass is intentionally omitted here despite cmake being run.
+# That eclass expects a "${S}/CMakeLists.txt" file during src_prepare(),
+# whereas this package uniquely generates one "${BUILD_DIR}/OCP/CMakeLists.txt"
+# file for each active Python version.
+inherit llvm multiprocessing python-r1
+
+MY_COMMIT="9a6210e3c490e741719ccdada900c70e58b09c91"
+OCCT_PV=$(ver_cut 1-3)
+
+DESCRIPTION="Python wrapper for OCCT generated using pywrap"
+HOMEPAGE="https://github.com/CadQuery/OCP"
+SRC_URI="https://github.com/CadQuery/OCP/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+KEYWORDS="~amd64 ~x86"
+SLOT="0"
+
+# This version requirement derives from "${BUILD_DIR}/OCP/CMakeLists.txt".
+BDEPEND=">=dev-util/cmake-3.16"
+RDEPEND="~sci-libs/opencascade-${OCCT_PV}[json,tbb,vtk]"
+DEPEND="${RDEPEND}
+	>=dev-python/cadquery-pywrap-${OCCT_PV}_rc0[${PYTHON_USEDEP}]
+"
+
+S="${WORKDIR}/OCP-${MY_COMMIT}"
+BUILD_DIR="${S}"
+
+# Ensure the path returned by get_llvm_prefix() contains clang.
+llvm_check_deps() {
+	has_version -r "sys-devel/clang:${LLVM_SLOT}"
+}
+
+# OCP currently requires manual configuration, compilation, and installation
+# inspired by the conda-specific "build-bindings-job.yml" file.
+src_prepare() {
+	default
```

Comment:
```suggestion
```
(Otherwise we apply the patches twice, cause `cmake_src_prepare` also does this)